snakePipes/workflows/RNA-seq/Snakefile

```diff
@@ -276,7 +276,8 @@ onstart:
 ################################################################################
 
 if not fromBAM:
-    localrules: FASTQ1, FASTQ2
+    localrules: FASTQ1, FASTQ2, Salmon_wasabi
```

Comment:
That might fix things randomly for us, but it's unlikely to do so for others.

Reply:
I'm aware of this, the issue is still open.